Backend/Backend/Models/Podcast.cs

```diff
@@ -1,56 +1,60 @@
 ﻿using System.ComponentModel.DataAnnotations;
-using System.ComponentModel.DataAnnotations.Schema;
 
 namespace Backend.Models;
 
 public class Podcast : BaseEntity
 {
-
-    public Podcast() 
-    {
-        Id = new Guid();
-        Name  = string.Empty;
-        Description = string.Empty;
-        Podcaster = null!;
-        PodcasterId = Guid.Empty;
-        Tags = Array.Empty<string>();
-        IsExplicit = false;
-        Type = PodcastType.Real;
-        Episodes = new List<Episode>();
-        AverageRating = 0;
-        TotalRatings = 0;       
-    }
-
+    /// <summary>
+    /// Podcast ID
```

Comment:
I think you mightve deleted the constructor for this

Reply:
I did, it was redundant because the properties are already being assigned a default value.